sherpa/plot/__init__.py

```diff
@@ -113,9 +122,108 @@
            'Confidence1D', 'Confidence2D',
            'IntervalProjection', 'IntervalUncertainty',
            'RegionProjection', 'RegionUncertainty',
+           'TemporaryPlottingBackend',
+           'set_backend',
```

Comment:
Not sure if I can do this with the GH interface, so I can only comment, but we need to add

    TemporaryPlottingBackend
    set_backend

to `docs/plots/plot.rst`